test/unit/test_cublas_tile/test_trsm.h

```diff
@@ -0,0 +1,100 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <cuda_runtime.h>
+#include <functional>
+#include <sstream>
+#include <tuple>
+#include "gtest/gtest.h"
+#include "dlaf/blas/enum_output.h"
+#include "dlaf/cublas/error.h"
+#include "dlaf/cublas_tile.h"
+#include "dlaf/matrix/copy_tile.h"
+#include "dlaf/matrix/tile.h"
+#include "dlaf/memory/memory_view.h"
+#include "dlaf_test/matrix/util_tile.h"
+#include "dlaf_test/matrix/util_tile_blas.h"
+#include "dlaf_test/util_types.h"
+
+namespace dlaf {
+namespace test {
+
+using namespace dlaf;
+using namespace dlaf::matrix;
+using namespace dlaf::matrix::test;
+using namespace dlaf::tile;
+using namespace testing;
+
+using dlaf::util::size_t::mul;
+
+template <typename T>
+void print_tile(Tile<T, Device::CPU> const& t) {
+  auto s = t.size();
+  auto n = s.cols();
+  auto m = s.rows();
+  for (int i = 0; i < m; ++i) {
+    for (int j = 0; j < n; ++j) {
+      std::cout << t(TileElementIndex{i, j}) << " ";
+    }
+    std::cout << std::endl;
+  }
+}
```

Comment:
Is this debug code supposed to me added to master?

Reply:
Nope, good catch!

Reply:
Removed.